raiden/tests/integration/api/rest/test_payments.py

```diff
@@ -300,7 +290,30 @@ def test_api_payments_with_hash_no_secret(
     )
     response = request.send().response
     assert_proper_response(response, status_code=HTTPStatus.CONFLICT)
-    assert payment == payment
+
+
+@raise_on_failure
+@pytest.mark.parametrize("number_of_nodes", [2])
+@pytest.mark.parametrize("enable_rest_api", [True])
+def test_api_payments_post_without_required_params(
+    api_server_test_instance, raiden_network, token_addresses
+):
+    _, app1 = raiden_network
+    token_address = token_addresses[0]
+
+    request = grequests.post(api_url_for(api_server_test_instance, "paymentresource",),)
```

Comment:
what is grequest?

Reply:
It's a gevent version of requests. To be honest, I thing just requests should be fine, but I just copied the test code from another.